apps/docs/src/components/SEO/ArticleMeta.tsx

```
@@ -0,0 +1,170 @@
+/**
+ * Article Meta Component
+ *
+ * Provides Open Graph and Twitter Card meta tags for rich share previews.
+ * Used on news articles and documentation pages.
+ */
+
+import React from "react";
+import Head from "@docusaurus/Head";
```

ESLint

[error] `@docusaurus/Head` import should occur before import of `react`

(import/order)